tests/load/script_archive_unused/deploy_with_genlayer_sdk.py

```
@@ -0,0 +1,367 @@
+    def get_contract_schema_for_code(self, contract_code):
+        """Get contract schema for validation"""
+        contract_hex = "0x" + contract_code.encode('utf-8').hex()
+        
+        request = {
+            "jsonrpc": "2.0",
+            "method": "gen_getContractSchemaForCode",
+            "params": [contract_hex],
+            "id": 1
+        }
+        
+        response = requests.post(self.rpc_url, json=request)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)